docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/front-end-enablement-service-guides.md

```diff
@@ -0,0 +1 @@
+This section contains how-to guides for individual tasks. 
```

Comment:
I assume spryker-cloud-commerce-os/front-end-enablement-service/building-a-front-end.md sits under this and will be joined by others?